homeassistant/components/rfxtrx/__init__.py

```diff
@@ -44,6 +44,7 @@
 CONF_DEBUG = "debug"
 CONF_OFF_DELAY = "off_delay"
 EVENT_BUTTON_PRESSED = "button_pressed"
```

Comment:
The event name should be prefixed with the domain name. We should probably make a breaking change later and fix this.

Reply:
Yup. Many of those coming up :). Trying to push all breaking changes to one step.

Reply:
https://github.com/home-assistant/core/pull/37565